src/sentry/incidents/subscription_processor.py

```diff
@@ -550,8 +551,29 @@ def trigger_alert_threshold(
         :return:
         """
         self.trigger_alert_counts[trigger.id] += 1
+
+        # If an incident was created for this rule, trigger type, and subscription
+        # within the last 10 minutes, don't make another one
+        last_incident: Incident | None = (
+            Incident.objects.filter(alert_rule=trigger.alert_rule).order_by("-date_added").first()
+        )
+        trigger_incidents = [incident.id for incident in trigger.triggered_incidents.all()]
+        last_incident_projects = (
+            [project.id for project in last_incident.projects.all()] if last_incident else []
+        )
+        minutes_since_last_incident = (
+            (timezone.now() - last_incident.date_added).seconds / 60 if last_incident else None
```

Comment:
@isabellaenriquez @wedamija should we use `date_started` instead of `date_added` here?

Reply:
when i had discussed this with dan we referenced `date_added` which i understand as being when the incident instance is actually created, but correct me if i'm wrong!